swarm/swarm.go

```diff
@@ -51,19 +57,29 @@ func run(cmd string) {
 }
 
 func getleaderHost() string {
```

Comment:
Sure, I might remove the function as well now swarm doesn't require etcd leader for service discovery.